app/controllers/schools/assignments_controller.rb

```diff
@@ -0,0 +1,18 @@
+module Schools
+  class AssignmentsController < ApplicationController
+    # PATCH /school/assignments/:id/update_milestone_number
+    def update_milestone_number
+      @target = Target.find(params[:id])
+      @page_no = params[:page] || 1
+
+      authorize(@target, policy_class: Schools::AssignmentPolicy)
```

Comment:
```suggestion
      target = authorize(Target.find(params[:id]), policy_class: Schools::AssignmentPolicy)
```
you can also do this